.taskcluster.yml

```diff
@@ -145,7 +145,7 @@ tasks:
                           git fetch ${repository} ${head_branch}
                           && git config advice.detachedHead false
                           && git checkout FETCH_HEAD
-                          && python automation/taskcluster/decision_task.py pull-request
+                          && python automation/taskcluster/decision_task.py raptor
```

Comment:
To be 100% explicit: this change should be reverted 🙂

Reply:
Hi Johan,
I reverted the changes in this commit: https://github.com/mozilla-mobile/fenix/pull/4430/commits/55a76b8aa47ca771022ed3cdafd5c999e80b6e49

I also did a rebase and now i see that i have 42 commits from other people. Let me know if i should cancel the pull request and start a new one.

Reply:
Yeah, it's probably easier to start over.